data/models/teleicare_history/ica_declaration.py

```diff
@@ -95,3 +97,29 @@ class IcaVersionDeclaration(models.Model):
     class Meta:
         managed = False
         db_table = "ica_versiondeclaration"
+
+
+class IcaPopulationCibleDeclaree(models.Model):
+    vrsdecl_ident = models.IntegerField(
+        primary_key=True
+    )  # The composite primary key (vrsdecl_ident, popcbl_ident) found, that is not supported. The first column is selected.
+    popcbl_ident = models.IntegerField()
+    vrspcb_popcible_autre = models.TextField(blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = "ica_population_cible_declaree"
+        unique_together = (("vrsdecl_ident", "popcbl_ident"),)
```

Comment:
Est-ce que c'est possible d'avoir plusieurs lignes avec le même `vrsdecl_ident` alors ? Je demande car "primary_key=True implies [null=False](https://docs.djangoproject.com/en/5.2/ref/models/fields/#django.db.models.Field.null) and [unique=True](https://docs.djangoproject.com/en/5.2/ref/models/fields/#django.db.models.Field.unique)." - https://docs.djangoproject.com/en/5.2/ref/models/fields/#primary-key
alors si il y a plusieurs qui sont les mêmes faut pas ajouter ligne 104 et 117 ?
Sinon, cette ligne `unique_together` est superflu

Reply:
La même remarque s'applique à `IcaPopulationRisqueDeclaree` et `IcaEffetDeclare` (donc ligne 130 aussi). 

Si je comprends bien Téléicare calcule un ID unique en concaténant l'ID de la déclaration et celui de l'objet (population, effet, etc), mais nous on prend ces IDs dans des champs séparés ?